dockstore-webservice/src/main/java/io/dockstore/webservice/permissions/PermissionsInterface.java

```diff
@@ -153,35 +184,6 @@ static List<Permission> mergePermissions(List<Permission> dockstoreOwners, List<
         return permissions;
     }
 
-    /**
-     * Returns the "original owners" of a workflow as list of {@link Permission}. The original owners
-     * are the <code>workflow.getUsers()</code>, e.g., who created the hosted entry, or members of the
-     * GitHub repo.
-     *
-     * @param workflow
-     * @return
-     */
-    static List<Permission> getOriginalOwnersForWorkflow(Workflow workflow) {
-        return workflow.getUsers().stream()
-                .map(user -> {
-                    // This is ugly in order to support both SAM and InMemory authorizers
-                    final User.Profile profile = user.getUserProfiles().get(TokenType.GOOGLE_COM.toString());
-                    if (profile != null && profile.email != null) {
-                        return profile.email;
-                    } else {
-                        return user.getUsername();
-                    }
-                })
-                .filter(email -> email != null)
-                .map(email -> {
-                    final Permission permission = new Permission();
-                    permission.setEmail(email);
```

Comment:
on the other hand, is it because we were already "wrongly" stuffing the other ids into the email field?